RELEASE.md

```diff
@@ -1,27 +1,21 @@
+# _Infrastructure - Dependency Update_
 
 _What is being released?_
 
-This release moves transaction-related components to a dedicated `legaldocumentation.transaction` namespace for transaction concepts, and all empty types to `legaldocumentation.transaction.additionalterms` namespace, in preparation for the ISDA Foundations migration.
-
-- Created new namespaces `legaldocumentation.transaction` and `legaldocumentation.transaction.additionalterms`
-- Moved type `TransactionAdditionalTerms` and the "additional terms" components that roll up to it to the new namespace provided they are not empty
-- Moved all empty types related to “additional terms” (used in TransactionAdditionalTerms) to sub-namespace legaldocumentation.transaction.additionalterms
-- Deleted unused namespace `legaldocumentation.master.additionalterms`
-
-
-_Backward-incompatible changes_
+This release updates the `DSL` dependency.
 
-None.
+Version updates include:
+- `DSL` 9.34.2 bug fix where removing duplicate import causes problems where those imports use aliases. For further details see DSL release notes: https://github.com/finos/rune-dsl/releases/tag/9.34.2
+- `DSL` 9.35.0 bug fix for recursive reporting rules and support for labels on circular types. For further details see DSL release notes: https://github.com/finos/rune-dsl/releases/tag/9.35.0
+- `DSL` 9.35.1 Rule source label fix and maintenance of generated serialization code. For further details see DSL release notes: https://github.com/finos/rune-dsl/releases/tag/9.35.1
+- `DSL` 9.36.0 Added condition support in typeAlias. For further details see DSL release notes: https://github.com/finos/rune-dsl/releases/tag/9.36.0
+- `DSL` 9.36.1 Use package name first segment for model name in RuneDataType annotation. For further details see DSL release notes: https://github.com/finos/rune-dsl/releases/tag/9.36.1
+- `DSL` 9.36.2 Fix extended rule source with extended type. For further details see DSL release notes: https://github.com/finos/rune-dsl/releases/tag/9.36.2
+- `DSL` 9.36.3 Fix XML serializer substitution groups. For further details see DSL release notes: https://github.com/finos/rune-dsl/releases/tag/9.36.3
+- `DSL` 9.36.4 Make XML serializer substitution groups work in a backward compatible manner. For further details see DSL release notes: https://github.com/finos/rune-dsl/releases/tag/9.36.4
+- `DSL` 9.36.5 Handle empty inputs when set on meta function output & Fix for setting meta on nested objects. For further details see DSL release notes: https://github.com/finos/rune-dsl/releases/tag/9.36.5
+- `DSL` 9.36.6 Fix multi cardinality nested meta. For further details see DSL release notes: https://github.com/finos/rune-dsl/releases/tag/9.36.6
 
-_Review Directions_
+_Review directions_
```

Comment:
Need a comment here stating that json expectations diffs are expected due to the added support for meta data in functions in DSL versions 9.36.6